scripts/variantstore/tieout/add_max_as_vqslod.py

```diff
@@ -19,23 +19,45 @@
 
         parts = line.split("\t")
         
-        if (parts[6] == "ExcessHet"):
+        if ("ExcessHet" in parts[6]):
             continue
+
+        if (len(sys.argv) > 2 and sys.argv[2] == "loose"):
```

Comment:
I didn't see this option in the readme. When should this be used?

Reply:
documented